docs/doxygen/overviews/xrc_format.h

```diff
@@ -731,6 +731,9 @@ wxAuiPaneInfo objects have the following properties:
     Sets the ideal size for the pane.}
 @row3col{floating_size, @ref overview_xrcformat_type_size,
     Sets the size of the floating pane.}
+@row3col{floating_client_size, @ref overview_xrcformat_type_size,
+    Sets the client size of the floating pane. This has precedence over
+    floating_size.}
```

Comment:
Sorry, a minor addition:
```suggestion
    floating_size. This attribute is available since wxWidgets 3.3.1.}
```

Reply:
Done.  See bb1a1942593def39c54c518d939c99ae04227b37